js/controller.js

```diff
@@ -314,9 +316,479 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.resetUpNextInfo(true);
       this.state.contentTree = contentTree;
       this.state.playerState = CONSTANTS.STATE.START;
+      this.onStyleFetched();//xenia: temporary here, for testing, remove before creating PR
       this.renderSkin({"contentTree": contentTree});
     },
 
+    onStyleFetched: function (event, thumbnails) {
+      thumbnails = //xenia: mock data, for testing with asset: RhbDFiMzE61s6Sv--ouyc4V5x3N3rS5C
+{
+   "data":{
+      "available_time_slices":[
+         0,
+         10,
+         20,
+         30,
+         40,
+         50,
+         60,
+         70,
+         80,
+         90,
+         100,
+         110,
+         120,
+         130,
+         140,
+         150,
+         160,
+         170,
+         180,
+         190,
+         200,
+         210,
+         220,
+         230,
+         240,
+         250,
+         260,
+         270,
+         280,
+         290,
+         300,
+         310,
+         320,
+         330
+      ],
+"available_widths":[
+         120,
+         240
+      ],
+      "thumbnails":{
```

Comment:
This may have already been addressed during yesterday's demo, but eventually this data will not be hard-coded, correct?